example/lib/main.dart

```diff
@@ -268,8 +268,25 @@ class _HomePageState extends State<HomePage> with WidgetsBindingObserver {
     Pspdfkit.setLicenseKey("LICENSE_KEY_GOES_HERE");
   }
 
+  void flutterPdfActivityOnPauseHandler() {
+    print("flutterPdfActivityOnPauseHandler");
+  }
+
+  void pdfViewControllerWillDismissHandler() {
+    print("pdfViewControllerWillDismissHandler");
+  }
+
+  void pdfViewControllerDidDismissHandler() {
+    print("pdfViewControllerDidDismissHandler");
+  }
```

Comment:
I'm fine with this, for now, but going forward, we need to consolidate events for iOS and Android. As a user if our Flutter library, having platform specific callbacks should only be necessary for platform specific features. Both the view controller dismissal, and the pause event, seem to serve a similar purpose (i.e. giving the developer a hook for when the PDF view is dismissed).

Reply:
We could do that here too. I initially went ahead with keeping them separate because iOS had two events (one before dismiss, one after dismiss), while Android had just just one - onPause. I was also a little confused about the naming though. What should we call the generic callback common between both platforms?